pkg/controller/cloneset/sync/cloneset_scale.go

```diff
@@ -70,10 +71,14 @@ func (r *realControl) Scale(
 	diffRes := calculateDiffsWithExpectation(updateCS, pods, currentRevision, updateRevision)
 	updatedPods, notUpdatedPods := clonesetutils.SplitPodsByRevision(pods, updateRevision)
 
+	if diffRes.scaleNum > 0 && diffRes.scaleUpLimit <= 0 {
```

Comment:
Shoud it be:
`if diffRes.scaleNum > 0 && diffRes.scaleUpLimit < diffRes.scaleNum {`

Reply:
done